pact/marmalade-util/util-v1.pact

```diff
@@ -1,11 +1,13 @@
 (namespace (read-msg 'ns))
 
 (module util-v1 GOVERNANCE
-  (use kip.token-policy-v2 [token-policies concrete-policy QUOTE_POLICY NON_FUNGIBLE_POLICY ROYALTY_POLICY COLLECTION_POLICY GUARD_POLICY])
+  (use kip.token-policy-v2 [ concrete-policy QUOTE_POLICY NON_FUNGIBLE_POLICY ROYALTY_POLICY COLLECTION_POLICY GUARD_POLICY])
+  (use marmalade.policy-manager )
 
   (defcap GOVERNANCE ()
     (enforce-guard (keyset-ref-guard 'marmalade-admin )))
 
+
   (defconst DEFAULT:object{concrete-policy}
```

Comment:
Do we still need the concrete-policy schema? Since we're taking in a list of strings we could do the translation from booleans to array of strings in the consuming code (FE)

If we want to keep it for convenience maybe also move the schema to the utils instead of in the kip.token-policy-v2 ?